src/main/java/com/powsybl/openloadflow/network/LfOverloadManagementSystem.java

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.openloadflow.network;
+
+import com.powsybl.openloadflow.network.impl.LfSwitch;
+
+import java.util.Objects;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
+ */
+public class LfOverloadManagementSystem {
+
+    private final LfBranch branchToMonitor;
```

Comment:
I think you must add a boolean that says if the automation system is enabled or not. It is quite typical for these systems.

Reply:
Done